src/components/infinite-scroll/infinite-scroll.ts

```diff
@@ -212,11 +222,14 @@ export class InfiniteScroll {
    * enabled or not. Setting to `false` will remove scroll event listeners
    * and hide the display.
    */
-  enable(shouldEnable: boolean) {
+  setEnable(shouldEnable: boolean) {
```

Comment:
Let's change this back to `enable` everywhere to match menu: https://github.com/driftyco/ionic/blob/master/src/components/menu/menu.ts#L538